apps/client/src/constants/index.ts

```diff
@@ -1 +1,15 @@
+import { ContentDensity } from '~/types';
+
 export const DEFAULT_LOCALE = 'en' as const;
```

Comment:
These are a mistake. No need to `as const` on const string declarations. It already narrows to the type to `en` instead of string. Same with of primitives. It only matters on rich data.